runner/mocks/poolManagerController.go

```diff
@@ -0,0 +1,200 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
This file can be automatically generated using a ```go generate``` stanza. You can add it in ```runner/interfaces.go``` after the import statement like so:


```go
//go:generate mockery --name=PoolManagerController
```

You can then run:

```bash
go generate
```

in that folder or:

```bash
go generate ./...
```

in the root folder of the project, and it will automatically generate/update the mock.